draft/authentication-basic.md

```diff
@@ -0,0 +1,67 @@
+---
+RFC: XXXX
+Author: Kyle Fuller
+Status: Draft
+Created: 2015-09-22
+Last Modified: 2015-09-28
+---
+
+# API Blueprint RFC XXXX: Basic Authentication Scheme
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Rationale](#rationale)
+- [Backwards Compatibility](#backwards-compatibility)
+
+## Abstract
+
+This RFC adds the "Basic" authentication scheme for the Authentication
+framework proposed in RFC XXXX.
+
+## Motivation
+
+Basic authentication is a commonly used authentication mechanism, and is part
+of the HTTP/1.1 specification defined in
+[RFC1945 section 11.1](http://tools.ietf.org/html/rfc1945#section-11.1).
+
+## Rationale
+
+The "Basic" authentication scheme is based on the model that the user
+agent must authenticate itself with a user-ID and a password.
+
+As such, a basic authentication scheme may configure two properties,
+`username` and `password`. These properties indicate a sample username and
+password that may be used.
+
+For example, as a named authentication scheme:
+
+```apib
+### Auth (Basic)
+
++ username: kyle
++ password: b2952d03bda09cb5f63b0162fbbee77c
+```
+
+As an anonymous scheme:
+
+```apib
++ Basic
+    + username: kyle
+    + password: b2952d03bda09cb5f63b0162fbbee77c
+```
+
+A client when making a request that is using the Basic authentication scheme
```

Comment:
I was trying to make it absolutely clear that any clients or tooling around API Blueprint that understands the semantic meaning around the basic authentication scheme should implement authentication as defined.

I'm happy to remove this section of the RFC draft if we all think this is clear without this section.